snafu/benchmarks/dnsperf/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry.centos.org/centos:8
+
+COPY snafu/image_resources/centos8-appstream.repo /etc/yum.repos.d/centos8-appstream.repo
+
+# dnsperf dependencies
+RUN dnf install -y --nodocs epel-release && \
+    dnf install -y --nodocs \
+        ck-devel \
+        openssl-devel \
+        yum-plugin-copr && \
+    dnf copr enable @dnsoarc/dnsperf -y && \
+    dnf install dnsperf -y && \
```

Comment:
Be consistent with your commands, also `-y` is a top level `dnf` flag:
`dnf -y install package`

Reply:
I'd make one mega `dnf install` rather than breaking them out.